src/platform/backends/qemu/dnsmasq_server.cpp

```diff
@@ -52,6 +52,13 @@ mp::DNSMasqServer::DNSMasqServer(const Path& data_dir, const QString& bridge_nam
     conf_file.open();
     conf_file.close();
 
+    QFile fd_Hosts(QDir(data_dir).filePath("dnsmasq.hosts"));
+    if (!fd_Hosts.exists())
+    {
+        fd_Hosts.open(QIODevice::WriteOnly);
+        fd_Hosts.close();
```

Comment:
This `close()` isn't necessary as [`QFile`'s dtor](https://doc.qt.io/qt-5/qfile.html#dtor.QFile) will close it automatically, but it's not wrong to have it either.